app/src/main/java/org/oppia/app/story/testing/StoryFragmentTestActivity.kt

```diff
@@ -19,14 +21,16 @@ class StoryFragmentTestActivity : InjectableAppCompatActivity(), RouteToExplorat
     storyFragmentTestActivityPresenter.handleOnCreate()
   }
 
-  override fun routeToExploration(explorationId: String, topicId: String?) {
+  override fun routeToExploration(internalProfileId: Int, topicId: String, storyId: String, explorationId: String) {
     // Do nothing since routing should be tested at the StoryActivity level.
   }
 
   companion object {
     /** Returns an [Intent] to create new [StoryFragmentTestActivity]s. */
-    fun createTestActivityIntent(context: Context, storyId: String): Intent {
+    fun createTestActivityIntent(context: Context, internalProfileId: Int, topicId: String, storyId: String): Intent {
       val intent = Intent(context, StoryFragmentTestActivity::class.java)
+      intent.putExtra(INTERNAL_PROFILE_ID_TEST_INTENT_EXTRA, internalProfileId)
+      intent.putExtra(TOPIC_ID_TEST_INTENT_EXTRA, topicId)
```

Comment:
ditto

Reply:
I agree with your point here. For this I have filed an issue: https://github.com/oppia/oppia-android/issues/747

Because this is something which is not a very high priority task. Also, once we finalise what convention we want to follow, it would can be `good-first-issue`.